### ISSUE TITLE
CNExT drug exposure extract has possible issues

### ISSUE THREAD

**Issue by lvarnedoe** [UNMCCC/CRIIS_Source_Extracts_for_OMOP#28]
The cnext_drug_exposure extract only pulls from the chemo table. Per OMOP description, the drug exposure table is to include a drug ingested or otherwise introduced into the body. Drugs include prescription and over-the-counter medicines, vaccines, and large-molecule biologic therapies, but not radiation.

**Comment by lvarnedoe** [comment]
Drug exposure query - some other potential issues:
PERSON_ID - pulled from PatExtended and returns only zeroes; value pulled from Hospital in visit details and visit occurrence - are the person_id values to be used to cross-reference result sets? 
VISIT_OCCURRENCE_ID - pulled from Patient. Pulled from Hospital in visit details and visit occurrence. Most of the records I checked have the same numbers, but not all. For cross-reference purposes, I would think this value should be the same across all result sets.
Condition "CHM.F04755 is not null" in predicate limits number of rows returned and is not in the Chemo visit details query-shouldn't they be the same?

**Comment by lvarnedoe** [comment]
Should text field (F05229) in chemo table be included in extract? It contains info on the drug administered, amount, and location received.

**Comment by lvarnedoe** [comment]
Inigo, can you reopen this issue. Misunderstood a button and closed it by mistake. Thx.

**Comment by lvarnedoe** [comment]
Never mind. I was able to reopen it myself.

**Comment by isangil** [comment]
can we close this one? seems now as duplicate.

**Comment by lvarnedoe** [comment]
No, it's not a duplicate. the drug exposure script only pulls from chemo and I think it should pull from every therapy table - hormone, immuno, other, etc... Doing so would mean duplicate data coming from various scripts, though. Just the first comment applies anymore. The issues in the 2nd comment are in separate issues.
